### ISSUE TITLE
[SUGGESTION] Option for Autototem/offhand to prioritize inventory items over hotbar

### ISSUE THREAD

**Issue by REM0VED** [pyroclient/pyro#251]
There needs to be an option for autotem/offhand to prioritize totems/gaps/crystals in inv over hotbar as right now it really fucks with the hotbar, dropping items and messing shit up. Also autototem needs to be way, way more reliable for Pyro to be a stand alone client, same with offhand switching at required health before you die.

**Comment by dotdivee** [comment]
Yeah thats pretty much the only reason im using Salhack's autototem over Pyro's rn.
